swagger.yaml

```diff
@@ -1,75 +1,164 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
+  description: "This API retrieves CORE permissions information on different users."
+  version: "1.0"
+  title: "CORE Permissions API"
+
+tags:
+- name: "permissions"
+  description: "HR, Student, and Finance permissions (1-5)"
+
 schemes:
-  - http
-basePath: /api/v0
+- "https"
+
+host: "api.oregonstate.edu"
+
+basePath: "/v1"
+
+externalDocs:
+  description: "GitHub Repository"
+  url: "https://github.com/osu-mist/core-permissions-api"
+
 consumes:
-  - text/plain
+  - "application/json"
+
 produces:
-  - text/plain
+  - "application/json"
+
 paths:
-  /:
+
+  /permissions/{osuID}:
```

Comment:
Could we change the endpoint from `/permissions` to something like `/core-permissions`? I just want to specify that these data permissions are specific to the CORE reporting application